app/src/components/market/common/outcome_table/index.tsx

```diff
@@ -90,18 +102,28 @@ export const OutcomeTable = (props: Props) => {
     withWinningOutcome = false,
     showPriceChange = false,
     showSharesChange = false,
+    newBonds,
   } = props
 
+  let winningBondIndex = -1
+  bonds.forEach((bond, bondIndex) => {
+    if ((winningBondIndex === -1 || bonds[winningBondIndex].bondedEth.lt(bond.bondedEth)) && bond.bondedEth.gt(0)) {
+      winningBondIndex = bondIndex
+    }
+  })
```

Comment:
@KadenZipfel I looked at it and it seems that it won't always evaluate to true but that checker is just used for determining if value is unset...and since that is forEach loop and "if" condition looks like this:(winningBondIndex||something)&&something it won't always evaluate to true...